src/getDeploy/getRpcInteractions.ts

```diff
@@ -29,15 +31,20 @@ const getEncodedArgs = async (
       async ([, dataArr]) => {
         const paramValueContainer = Array(dataArr.length)
         const paramTypeContainer = Array(dataArr.length)
-        for (const argName in userModuleArgs) {
-          // find index in config
-          const idx = dataArr.findIndex((config) => config.name === argName)
-          // if index is found
-          if (idx >= 0) {
-            // put arg in the correct idx in param container
-            paramValueContainer[idx] = userModuleArgs[argName]
-            // put type in the correct idx in type container
-            paramTypeContainer[idx] = { type: dataArr[idx].type }
+
+        for (let i = 0; i < dataArr.length; i++) {
```

Comment:
Iterating over the config data rather than over the user args makes it easier to identify tuples